### PR TITLE
MINOR - Adding New York Times logo to streams page

### PR DESCRIPTION
**Pull request by manjuapu**

### CODE REVIEW

**Comment by manjuapu** [comment]
@guozhangwang Please review. Thanks!!

File: docs/streams/index.html [hunk 1]
```diff
@@ -205,35 +205,37 @@ <h3 class="customer-title">See how Kafka Streams is being used</h3>
     <div class="customer__cards">
          <div class="customer__card">
              <div class="customer__card__icon">
+            <img src="/{{version}}/images/icons/NYT.jpg" style="max-width:35%">
+          </div>  
+          <span class="customer__card__label">The New York Times uses Kafka and the Kafka Streams API to store and distribute, in real-time, published content to the various applications and systems that make it available to the readers.</span>
```
Comment (guozhangwang) [inline docs/streams/index.html]: nit: ... Apache Kafka and the Kafka Streams ...

**Comment by manjuapu** [comment]
Thanks @guozhangwang changed it now.

**Comment by guozhangwang** [comment]
Merged to trunk.

**Pull request closed without merging**